src/graph/client/graphClient.ts

```diff
@@ -11,7 +11,7 @@
  */
 
 declare let d3: any;
-
+/* tslint:disable */
```

Comment:
This disables all lint, right? I'd rather you disabled just certain rules.  And it should be at the top of the file.

Reply:
Had a second look at it. I'll remove it since I won't need it at this moment.